projects/plugins/jetpack/extensions/blocks/ai-assistant/edit.js

```diff
@@ -139,11 +139,40 @@ export default function AIAssistantEdit( { attributes, setAttributes, clientId }
 	// Content is loaded
 	const contentIsLoaded = !! attributes.content;
 
-	const handleAcceptContent = () => {
-		replaceBlocks(
-			clientId,
-			rawHandler( { HTML: markdownConverter.render( attributes.content ) } )
-		);
+	const getLastEditableElement = newContentBlocks => {
+		let lastEditableElement = null;
+
+		newContentBlocks.forEach( block => {
+			const element = document.querySelector( `.wp-block[data-block="${ block.clientId }"]` );
+			if ( element.contentEditable === 'true' ) {
```

Comment:
Is it possible that `element` is undefined? Maybe `element.contentEditable` can trigger an exception.